js/utilities.js

```diff
@@ -140,6 +140,154 @@ var ERMrest = (function(module) {
     };
 
     /**
+     * @function
+     * @param {Object} value A boolean value to transform
+     * @param {Object} [options] Configuration options
+     * @return {string} A string representation of a boolean value
+     * @desc Formats a given boolean value into a string for display
+     */
+    module._printBoolean = function(value, options) {
+        options = (typeof options === 'undefined') ? {} : options;
+        if (value === null) {
+            return '';
+        }
+        // TODO: What kinds of options are we supporting?
+        return Boolean(value).toString();
+    }
+
+    /**
+     * @function
+     * @param {Object} value An integer value to transform
+     * @param {Object} [options] Configuration options
+     * @return {string} A string representation of value
+     * @desc Formats a given integer value into a whole number (with a thousands
+     * separator if necessary), which is transformed into a string for display.
+     */
+    module._printInteger = function(value, options) {
+        options = (typeof options === 'undefined') ? {} : options;
+        if (value === null) {
+            return '';
+        }
+
+        // Remove fractional digits; implicitly transform value to number type
+        // TODO: Truncate or round?
+        value = Math.trunc(value);
```

Comment:
I think it is customary to round a floating point value to integer if converting. You might think of it as printing a floating point with zero fractional digits: `printf("%.0f", 5.55555)` will result in `6`.